### DISCUSSION TITLE
Navic lualine section

### DISCUSSION THREAD

**Discussion by niveksoft** [LazyVim/LazyVim#1722] [Q&A] [answered]
Is there anyway I can change the color of this section to match the rest? I have my team configured to use transparent background but this section is not respecting that.

Also how do I go about disabling this if needed?

![lualine](https://github.com/LazyVim/LazyVim/assets/26445418/7a4c5fbe-139e-4090-a5eb-462c84dd34fc)

**Comment by niveksoft** [comment, marked as answer]
I was able to disable it by doing. But would prefer keeping it and have it match my theme.

`return {
  "nvim-lualine/lualine.nvim",
  opts = function(_, opts)
    table.remove(opts.sections.lualine_c, 4)
  end,
`

In colorscheme.lua (I use catppuccin), I had to override the navic theme integration as follows
```
      integrations = {
        navic = {
          enabled = true,
          custom_bg = "NONE",
        },
```

**Reply by dpetka2001** [reply]
For having it match your theme, you can take a look at the specific section of my [dotfiles](https://github.com/dpetka2001/dotfiles/blob/main/dot_config/nvim/lua/plugins/colorscheme.lua).

**Reply by dpetka2001** [reply]
But this is specific to tokyonight colorscheme. You should do something similar if you use a different colorscheme, depending on which highlight groups your colorscheme uses for navic.

**Reply by niveksoft** [reply]
Awesome thanks! Looking at your config I was able to figure it out. I use catppuccin, which has navic configured, I just needed to set the custom_bg to NONE so it matches.

**Comment by dpetka2001** [comment]
See this [comment](https://github.com/LazyVim/LazyVim/discussions/116#discussioncomment-4763452) for transparency if you're using `tokyonight` colorscheme, otherwise you should consult your colorschemes docs for how to achieve transparency. You should also make sure your terminal is also transparent. About disabling, do you mean how to disable the specific navic lualine component or the navic plugin as a whole?
